core/src/test/java/org/elasticsearch/search/builder/SearchSourceBuilderTests.java

```diff
@@ -574,25 +305,4 @@ public void testEmptyQuery() throws IOException {
         String query = "{ \"query\": {} }";
         assertParseSearchSource(builder, new BytesArray(query), ParseFieldMatcher.EMPTY);
     }
-
-    public void testSearchRequestBuilderSerializationWithIndexBoost() throws Exception {
```

Comment:
Nevermind, just saw that `searchSourceBuilder.indexBoost` is now randomized by default in `createSearchSourceBuilder()createSearchSourceBuilder()` so I guess thats the reason why. Correct me if I'm wrong.